app/assets/javascripts/controllers/container_topology/container_topology_controller.js

```diff
@@ -141,18 +143,49 @@ function ContainerTopologyCtrl($scope, $http, $interval, $location, topologyServ
       return self.dblclick(d);
     });
 
+    added.append("image")
+      .attr("xlink:href", function (d) {
+        switch(d.item.kind) {
+          case 'ContainerManager':
+              return icons[d.item.display_kind];
+            //return 'vendor-'+ d.item.display_kind + ".svg";
+          default:
+            return null;
+        }
+      })
+      .attr("height", function(d) {
+         return 40;
+      })
```

Comment:
@martinpovolny [yes we can](http://1.bp.blogspot.com/_julY3XAJ89c/SWp6Glfyi1I/AAAAAAAAAXM/yUtCgwCG2Ac/s400/Yes+We+Can.jpg)